apps/services/auth-public-api/src/environments/environment.ts

```diff
@@ -51,7 +51,7 @@ const prodConfig = {
   },
   auth: {
     audience: '@island.is',
-    issuer: process.env.IDENTITY_SERVER_ISSUER_URL,
+    issuer: process.env.IDENTITY_SERVER_ISSUER_URL ?? '',
```

Comment:
Of course, good catch.
The reason why I added this is that when running tests `ts-jest` logs two warnings about ts errors because `AuthConfig` does not acceppt `string | undefined` for  `issuer`.  But should definitely solve that differently.